src/test/java/com/bluesource/projects/Show_Rate_on_Project_Show_Page_Verify.java

```diff
@@ -0,0 +1,76 @@
+package com.bluesource.projects;
+
+import com.orasi.bluesource.Accounts;
+import com.orasi.bluesource.Header;
+import com.orasi.bluesource.LoginPage;
+import com.orasi.utils.Constants;
+import com.orasi.utils.TestReporter;
+import com.orasi.utils.dataProviders.ExcelDataProvider;
+import com.orasi.web.WebBaseTest;
+import org.testng.ITestContext;
+import org.testng.annotations.*;
+
+public class Show_Rate_on_Project_Show_Page_Verify extends WebBaseTest {
+	//*****************
+	// Data Provider
+	//*****************
+	@DataProvider(name = "show_project_rate_verify", parallel = true)
+	public Object[][]scenarions(){
+		return new ExcelDataProvider(Constants.EXCEL_SHEETS + "/ShowRateOnProjectVerify.xlsx","Sheet1").getTestData();
+	}
+
+	@BeforeMethod
+	@Parameters({ "runLocation", "browserUnderTest", "browserVersion",
+			"operatingSystem", "environment" })
+	public void setup(@Optional String runLocation, String browserUnderTest,
+					  String browserVersion, String operatingSystem, String environment) {
+		setApplicationUnderTest("BLUESOURCE");
+		setBrowserUnderTest(browserUnderTest);
+		setBrowserVersion(browserVersion);
+		setOperatingSystem(operatingSystem);
+		setRunLocation(runLocation);
+		setEnvironment(environment);
+		setThreadDriver(true);
+		testStart("Show_Rate_on_Project_Show_Page_Verify");
+	}
+
+	@AfterMethod
+	public void close(ITestContext testResults){
+		endTest("TestAlert", testResults);
+	}
+
+	@Test(dataProvider = "show_project_rate_verify")
+	public void Show_Rate_on_Project_Show_Page_Verify(String scenario, String account, String project, String role)
```

Comment:
small java convention here but all methods start with lower case camel case, and you typically don't use a lot of underscores.  showRateOnProjectShowPageVerify.  Typically underscores are only used for constants

"If your variable stores a constant value, such as static final int NUM_GEARS = 6, the convention changes slightly, capitalizing every letter and separating subsequent words with the underscore character. By convention, the underscore character is never used elsewhere."  
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html